src/AI_Sideloader/AI.Sideloader.ListLoader.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+
+namespace Sideloader.ListLoader
+{
+    internal static partial class Lists
+    {
+        // internal main game furniture data
+
+        /// <summary>
+        /// Currently only resolving
+        /// </summary>
+        /// <param name="stream"></param>
```

Comment:
Remove any empty parts of docs so there's a warning if it does need to be filled in